scss/bootstrap/_variablesV5.scss

```diff
@@ -6,7 +6,7 @@
 // Color system
 
 // scss-docs-start gray-color-variables
-$white:    #fff !default;
+$white: #fff !default;
```

Comment:
J'ai un peu peur du rebase sur ce fichier !

Reply:
J'ai lancé la commande npm stylelint pour corriger les erreurs de la CI qu'on aura aussi dans la PR initiale, tu peux tout reprendre safely de ce fichier _variablesV5.scss qui est la plus up to date 

"stylelint": "stylelint assets/styles --fix",